systemd/timesyncd/config.sls

```diff
@@ -0,0 +1,59 @@
+include:
+  - systemd.reload
+
+{% from "systemd/map.jinja" import systemd with context -%}
+{% from "systemd/libtofs.jinja" import files_switch with context -%}
+
+{%- set timesyncd = systemd.get('timesyncd', {}) %}
+{%- set timezone = timesyncd.get('timezone', 'UTC') %}
+{%- set config = timesyncd.get('config', {}) %}
+
+{%- set virtual = salt['grains.get']('virtual') | default('physical', True) -%}
+{%- set virtual_subtype = salt['grains.get']('virtual_subtype') | default('', True) -%}
```

Comment:
`-%}` => `%}` to prevent problems with "double" whitespace control.

Reply:
fix it